History.md

```diff
@@ -25,6 +25,7 @@ This is the first Express 5.0 alpha release, based off 4.10.1.
     - `req.acceptsLanguage` - use `req.acceptsLanguages`
     - `res.json(obj, status)` signature - use `res.json(status, obj)`
     - `res.jsonp(obj, status)` signature - use `res.jsonp(status, obj)`
```

Comment:
No, because that is the change log for the first alpha. You can't changed what has already been released.